src/server/location.rs

```diff
@@ -43,7 +56,7 @@ impl LocationResult {
             dbg!("Found Google Header");
             return Self::from_headervalue(header);
         }
-        Self::default()
+        Self::from(settings)
```

Comment:
doh, we both missed this: It sounds like the LB's going to give us the header regardless of lookup failure, we just get empty values:

["The load balancer expands variables to empty strings when it cannot determine their values."](https://cloud.google.com/load-balancing/docs/custom-headers#variables)

Reply:
ok, added a check for blank google headers. Also added a test for it.